install.bash

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+yarn install --frozen-lockfile
+
+for dir in packages/*; do
+    echo "Installing... $dir"
+    pushd "$dir"
+    yarn install --frozen-lockfile
+    popd
+    echo "Done..."
+    echo ""
+done
+
+for dir in platform/*; do
+    echo "Installing... $dir"
+    pushd "$dir"
+    yarn install --frozen-lockfile
+    if [ -d "internal_pkgs" ]; then
+        for subdir in internal_pkgs/*; do
+            pushd "$subdir"
+            yarn install --frozen-lockfile
+            if [ "$(jq -r '.scripts.build' package.json)" != "null" ]; then
+                NODE_ENV=production yarn build
+            fi
+            popd
+        done
+    fi
+    popd
+    echo "Done..."
+    echo ""
+done
+
+NX_REJECT_UNKNOWN_LOCAL_CACHE=0 yarn nx run-many --target=build --verbose
+
+for dir in platform/*; do
+    if [ "$dir" == "platform/wab" ]; then
+        continue
+    fi
+    echo "Building... $dir"
+    pushd platform/"$dir"
```

Comment:
I got an error at this line:

```sh-session
Building... platform/canvas-packages
./install.bash: line 42: pushd: platform/platform/canvas-packages: No such file or directory
```

Since `$dir` will already be prefixed with `"platform/"`, I think this should be changed to:

```
    echo "Building... $dir"
    pushd "$dir"
```